### PR TITLE
New support doc--view tickets

### PR DESCRIPTION
**Pull request by wcpines**
@fancyremarker for review

@chasballew: wrapping up #61

### CODE REVIEW

File: source/topics/troubleshooting/view-open-support-tickets.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Sign in/up at [https://aptible.zendesk.com/hc/en-us/requests](https://aptible.zendesk.com/hc/en-us/requests). You can also request permission to view all of your organization's tickets by [sending us a note](https://support.aptible.com/contact/).
+
+If you submitted a ticket via [support@aptible.com](support@aptible.com), follow the link at the bottom of the email to see all open tickets ("_You can view this and your other tickets on the web_")
+
+NB: If you are submitting a ticket for the first time, you'll need to create a password in order to login to the zendesk account we've automatically generated for you.
```
Comment (fancyremarker) [inline source/topics/troubleshooting/view-open-support-tickets.md:5]: s/zendesk/Zendesk/ and consider replacing "NB: If" with "Note that if". We haven't adopted a language style guide so I'm fine leaving NB as is.

**Comment by fancyremarker** [comment]
This is good, thanks! See my one comment. Also, you will need to add this topic article to the TOC, which is a YAML file: https://github.com/aptible/support/blob/master/data/topics.yml

I _think_ this would go in the "Your Aptible Account" section, but you can decide.

If you think it would be helpful for future contributors, please also add a section to the README describing the steps for adding a new topic article.

**Comment by wcpines** [comment]
Sounds good. And I'll learn to spell Zendesk correctly, I promise :P

**Comment by wcpines** [comment]
Closing in favor of #92

**Pull request closed without merging**